multiversx_sdk_network_providers/api_network_provider.py

```diff
@@ -118,14 +122,47 @@ def get_transaction(self, tx_hash: str) -> TransactionOnNetwork:
 
         return transaction
 
-    def get_transactions(self, address: IAddress, pagination: IPagination = DefaultPagination()) -> List[TransactionOnNetwork]:
+    def get_account_transactions(self, address: IAddress, pagination: IPagination = DefaultPagination()) -> List[TransactionOnNetwork]:
         url = f"accounts/{address.bech32()}/transactions?{self._build_pagination_params(pagination)}"
         response = self.do_get_generic_collection(url)
 
         transactions = [TransactionOnNetwork.from_api_http_response(tx.get("txHash", ""), tx) for tx in response]
 
         return transactions
 
+    def get_transactions(self, tx_hashes: List[str], with_block_info: bool = True, with_results: bool = True) -> List[TransactionOnNetwork]:
```

Comment:
`get_bunch_of_transactions()` or something similar, to avoid ambiguity with the previous functionality of `get_transactions`? (optional).

Reply:
Renamed to `get_bunch_of_transactions()`